tests/sentry/api/endpoints/notifications/test_notification_actions_index.py

```diff
@@ -102,7 +128,7 @@ def test_get_with_queries(self, mock_trigger_types):
             trigger_type=1,
         )
 
-        query_data = {
+        query_data: Mapping[str, _QueryResult] = {
```

Comment:
generally I'd recommend a concrete type (`dict`) instead of `Mapping` -- usually just easier and you don't need the extra import